src/errors.ts

```diff
@@ -0,0 +1,34 @@
+export class InvalidStoreError extends Error {
```

Comment:
We should probably have a licences here :thinking:

Reply:
_resolved_ - thanks!

Reply:
Forgot about the license since it wasn't there previously, should be good now~